code/Base/StateSystem/PredictionStateAttribute.cs

```diff
@@ -0,0 +1,11 @@
+namespace Sports.StateSystem;
+
+[AttributeUsage( AttributeTargets.Class )]
+public class PredictStatesAttribute : Attribute
```

Comment:
I wonder if we can name this something better. It's not super self explanatory of what this really does unless you know the system

Reply:
PredictNextStates? CacheStates? CachePredictStates?

i had trouble finding names too...

Reply:
it's fine for now it's as simple as just renaming it down the line